packages/paste-core/components/minimizable-dialog/src/MinimizableDialogContent.tsx

```diff
@@ -19,7 +19,6 @@ const MinimizableDialogContent = React.forwardRef<HTMLDivElement, MinimizableDia
         {...safelySpreadBoxProps(props)}
         element={element}
         ref={ref}
-        margin={minimized ? 'space0' : 'space70'}
```

Comment:
question: why is that we got rid of this spacing?

Reply:
Made an margin between the chat composer/chat log and the dialog
<img width="433" alt="Screen Shot 2022-11-17 at 12 28 30 PM" src="https://user-images.githubusercontent.com/8482334/202553072-d7df5406-eb4a-435b-bbe0-a3c9a2094747.png">